package.json

```diff
@@ -348,8 +348,8 @@
     "classnames": "2.2.5",
     "dedent": "^0.7.0",
     "delete-empty": "^2.0.0",
-    "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.9.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.13.1",
```

Comment:
I saw you have another PR out for upgrading these, but the PRs have some slight differences. I'd like to get the other PR merged before this one, so the right people can review that dependency upgrade.